### PR TITLE
Add `CopyCode` operation to InteractiveWindow

### PR DESCRIPTION
**Pull request by genlu**
`CopyCode` copies code from user inputs (if any) to clipboard, standard-input is not considered code here: If selection is empty, then copy code from user inputs in current line, otherwise copy code from user inputs in all selected text.

A button for this operation is added to VS with a shortcut "ctrl + shift + c" in Interactive Window scope. The only other command that uses the same shortcut is `View.ClassView` in global scope.

copyinputs

This provides (a renamed) "copy inputs" command as described in #5212

@dotnet/roslyn-interactive

### CODE REVIEW

**Comment by genlu** [comment]
FYI @dotnet/roslyn-ide for button/shortcut in VS related change.

File: src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs [hunk 1]
```diff
@@ -2796,6 +2790,37 @@ public void Copy()
...
+            public void CopyInputs()
+            {
+                NormalizedSnapshotSpanCollection spans;
+                if (TextView.Selection.IsEmpty)
+                {
+                    spans = GetSpansFromCurrentLine();
+                }
+                else
+                {
+                    spans = GetSpansFromSelectedLines();
```
Comment (genlu) [inline src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs]: @amcasey suggests we stick with the behavior of regular copy here, i.e. only copy from selected text. I will change this unless someone has a better argument :)

**Comment by genlu** [comment]
ping... @dotnet/roslyn-interactive

File: src/InteractiveWindow/Editor/IInteractiveWindowOperations2.cs [hunk 2]
```diff
@@ -10,6 +10,12 @@ public interface IInteractiveWindowOperations2 : IInteractiveWindowOperations
         void Copy();
 
         /// <summary>
+        /// Copies only user inputs to clipboard.
```
Comment (amcasey) [inline src/InteractiveWindow/Editor/IInteractiveWindowOperations2.cs]: From the name and description, it's not clear whether standard input will be copied.

**Comment by amcasey** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
Late to the party but :+1: if you wanted an IDE review on the command stuff.

**Pull request merged**